core/util.go

```
@@ -70,7 +72,12 @@ func B64dec(x string) ([]byte, error) {
 
 func RandomString(byteLength int) string {
 	b := make([]byte, byteLength)
-	_, _ = rand.Read(b) // NOTE: Ignoring errors
+	_, err := io.ReadFull(rand.Reader, b) // NOTE: Ignoring errors
```

Remove "ignoring errors" comment